libcloudforensics/providers/aws/forensics.py

```diff
@@ -206,9 +206,24 @@ def StartAnalysisVm(
   Returns:
     Tuple[AWSInstance, bool]: a tuple with a virtual machine object
         and a boolean indicating if the virtual machine was created or not.
+
+  Raises:
+    RuntimeError: If the requested provider or function is not
+        implemented.
   """
+
   aws_account = account.AWSAccount(
       default_availability_zone, aws_profile=dst_profile)
+
+  # If no AMI ID is given we use the default Ubuntu 18.04
+  # in the region requested.
+  if not ami:
+    qfilter = [{'Name':'name', 'Values':[UBUNTU_1804_FILTER]}]
+    ami_list = aws_account.ListImages(qfilter)
+    if len(ami_list) > 1:
+      raise RuntimeError('error - ListImages returns >1 AMI image!')
```

Comment:
I'm not sure I understand this error message. Why are we expecting exactly one result? Maybe add a short comment to make it clear